python-package/securexgboost/core.py

```diff
@@ -2520,18 +2525,20 @@ def init_client(remote_addr=None, user_name=None,
     _CONF["nonce_ctr"] = 0 
 
 
-def init_server(enclave_image=None, log_verbosity=0):
+def init_server(enclave_image=None, client_list=[], log_verbosity=0):
     """
     Launch the enclave from an image. This API should be invoked only by the servers and not the clients.
 
     Parameters
     ----------
     enclave_image: str
         Path to enclave binary
+    client_list: list
+        List of client usernames (strings) allowed to use the enclaves
```

Comment:
Let's change this `client_list` description to `List of client usernames (strings) **in the collaboration** allowed to use the enclaves`